pingrid.py

```diff
@@ -793,3 +794,23 @@ def load_config(colon_separated_filenames):
         with open(fname) as f:
             config = deep_merge(config, yaml.safe_load(f))
     return config
+
+
+def target_range_format(leads_values,leads_keys,start_date,period_length):
```

Comment:
There is a version of this in predictions.py, that deals with one entry at a time. And the app seems to be using the predictions.py one. So maybe this one can go?